metadata.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "puppetlabs-secure_env_vars",
+  "version": "0.1.0",
+  "author": "Puppet",
+  "summary": "Run a command or script with sensitive environment variables.",
+  "license": "Apache-2.0",
+  "source": "https://github.com/puppetlabs/puppetlabs-secure_env_vars",
+  "dependencies": [
```

Comment:
We should list puppetlabs-stdlib here if we're using parsejson

Reply:
Added as a dependency. Not really sure how open to be on the version requirement, so I just picked `>= 6.0.0`.